### PR TITLE
Add IO benchmark in CI

### PR DESCRIPTION
**Pull request by jumaffre**
So that we can track IO performance of different platforms.

Benchmark takes ~2 seconds:

SGX:
```
1024000+0 records in
1024000+0 records out
1048576000 bytes (1.0 GB, 1000 MiB) copied, 2.10768 s, 498 MB/s
```

SEV-SNP ACI:
```
1024000+0 records in
1024000+0 records out
1048576000 bytes (1.0 GB, 1000 MiB) copied, 2.50719 s, 418 MB/s
```

### CODE REVIEW

File: .azure-pipelines-templates/common.yml [hunk 1]
```diff
@@ -11,6 +11,10 @@ jobs:
           sudo rm -rf build
         displayName: "Cleanup"
 
+      - script: | 
+          dd if=/dev/zero of=/tmp/zero bs=1k count=1000k
```
[the remark this answers is not in the document]
Comment (MahatiC) [reply .azure-pipelines-templates/common.yml:15]: Okay. Although from my understanding there shouldn't be any significant difference between the two platforms (unless the storage disks used are very different)

[the remark this answers is not in the document]
Comment (jumaffre) [reply .azure-pipelines-templates/common.yml:15]: Yes, it doesn't look like it from the measurements on this PR so I'm not sure it's actually worth merging to `main`.

[the remark this answers is not in the document]
Comment (MahatiC) [reply .azure-pipelines-templates/common.yml:15]: I'm fine either way given this check isn't eating up time.

**Comment by jumaffre** [comment]
Closing this as the results are consistent between SEV-SNP and SGX.

**Pull request closed without merging**